Analysis/Tasks/PWGUD/upcForward.cxx

```diff
@@ -59,13 +61,27 @@ struct UPCForward {
     bool ispositive = kFALSE;
     bool isnegative = kFALSE;
     /*this code below is suggested by evgeny.
-    this code is now hardcoded for run 246392 as we are not sure if trigger id is same for all the runs*/
-    uint64_t classIndexMUP11 = 54; // 246392
-    bool isMUP11fired = bc.triggerMask() & (1ull << classIndexMUP11);
+    this code is now hardcoded for runs  246391, 246392 for CMUP11
+    and 244980, 244982, 244983, 245064, 245066, 245068 for CMUP10*/
+    uint64_t classIndexMUP = -1;
+    Int_t iRunNumber = bc.runNumber();
+
+    if (iRunNumber == 246391 || iRunNumber == 246392) {
+      classIndexMUP = 51; //CMUP11
```

Comment:
Shouldn't this come from the physics selection instead of hard-coding it here?
@ekryshen What do you think?

Reply:
@jgrosseo Indeed, this is a temporary workaround. The problem is that the physics selection is running over collisions at the moment while here we need bc-joinable trigger alias decisions etc. I will implement it soon.

Reply:
OK